markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+              \item WP: Working Paper
+              \item PP: Published Paper (not paper, not article, not Article)
+            \end{itemize}
+          \item Title: Title of the paper
+          \item Authors: Authors: all comma separated; first name, last name. Patty Smith, Patty Smith, Petty Smith
+          \item Description: \textcolor{red}{Add kind and No of external working paper if applicable}
+            \begin{lstlisting}[breaklines]
+                Toulouse School of Economics Working Papers No 1391
+            \end{lstlisting}
+          \item DOI: YES (if it exists)
+          \item URL: YES (Editor displays an error at https, but it is accepted)
+        \end{itemize}
+    \item Language: English, EN
+\end{enumerate}
+
+## Data
+Sufficient are
+\begin{enumerate}
+    \item Data statement:\footnote{ Description: An overall statement on the data used in the project. A spearate field is provided to list and document the origin and key characteristics of the datasets}. For external packages, the following rule:
+        \begin{itemize}
+          \item If all data is public:
+            \begin{itemize}
+            \item Fill in: "All data is public and contained in the (external)         reproducibility package."
+            \item Add data availability statement if available from journal package; often separate from the zip, has to be copied over manually.
+            \end{itemize}
+          \item If some data is confidential: Paste from the readme notes about confidentiality.
+        \end{itemize}
+    \item Datasets:\footnote{Description of this field: "This field is used to provide an itemized list of datasets used in the project. The data are not documented here (specific metadata are available for documenting data of different types, like the DDO for microdata, the ISO 19139 for geographic datasets etc)"} Not used for PP, will be used for WP, if they data sets are of proper value and stored in another WB repository (like microdata library). \textcolor{red}{No such case arose so far. TBD}
+        \begin{enumerate}
+          \item Name
+          \item Data set ID: (empty)
+          \item Note: (empty)
+          \item Access policy:
+            \begin{itemize}
+              \item Published elsewhere: Note access/license
+              \item Published with package: Authors must confirm there is no confidentiality.
+            \end{itemize}
+          \item License (empty)
+          \item License URL (empty)
+          \item Data URL
+        \end{enumerate}
+\end{enumerate}
+
+## Methods, software and scripts
+Fields that are not listed can be discarded.
+\begin{enumerate}
+  \item Methods: Econometric methods used. \textcolor{red}{This field will learn from past entries.} Possibly add notes (more details about the context)
+  \item Software: Stata 17.0, R, etc
+  \item Scripts:
+      \begin{enumerate}
+          \item File name (WP+PP): Name of the zip file. Naming convention: PrimaryID\_prg\_v01.zip
```

Comment:
(WP+PP) - in other parts of the protocol, i assumed that fields that are not specified as either WP or PP apply to both. confusing to start to label (WP+PP) here; either need to do it consistently across full protocol or not at all. my suggestion would be to default that the protocol applies to all and specify when there are deviations.

Reply:
naming convention: why "_prg"?

Reply:
For program. When adding external resources, I cannot choose package, which would be more fitting. The closest fit was program. Thus for consistency

Reply:
got it.